package.json

```diff
@@ -14,7 +14,7 @@
   "unpkg": "dist/browser/index.global.js",
   "exports": {
     "node": {
-      "module": "./dist/node/index.mjs",
+      "import": "./dist/node/index.mjs",
```

Comment:
```suggestion
      "import": "./dist/node/index.mjs",
      "module": "./dist/node/index.mjs",
```

we want to keep `module` around in addition (there are some tools and bundlers that can pick up `module` but not `import`)

Reply:
No worries, wasn't aware, done :)